src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -505,6 +510,72 @@ def _test_log_file_mgmt(self, resource_group, server, database_engine):
 
         self.assertIsNotNone(result[0]['name'])
 
+    def _test_private_link_resource(self, resource_group, server, database_engine, group_id):
+        result = self.cmd('{} server private-link-resource list -g {} -s {}'
+                          .format(database_engine, resource_group, server)).get_output_in_json()
+        self.assertEqual(result[0]['properties']['groupId'], group_id)
+
+    def _test_private_endpoint_connection(self, resource_group, server, database_engine):
+        loc = 'eastus'
+        vnet = self.create_random_name('cli-vnet-', 24)
+        subnet = self.create_random_name('cli-subnet-', 24)
+        pe = self.create_random_name('cli-pe-', 24)
+        pe_connection = self.create_random_name('cli-pec-', 24)
+
+        # Prepare network
+        self.cmd('network vnet create -n {} -g {} -l {} --subnet-name {}'
+                 .format(vnet, resource_group, loc, subnet),
+                 checks=self.check('length(newVNet.subnets)', 1))
+        self.cmd('network vnet subnet update -n {} --vnet-name {} -g {} '
+                 '--disable-private-endpoint-network-policies true'
+                 .format(subnet, vnet, resource_group),
+                 checks=self.check('privateEndpointNetworkPolicies', 'Disabled'))
+
+        # Get Server Id and Group Id
+        result = self.cmd('{} server show -g {} -n {}'
+                          .format(database_engine, resource_group, server)).get_output_in_json()
+        server_id = result['id']
+        result = self.cmd('{} server private-link-resource list -g {} -s {}'
+                          .format(database_engine, resource_group, server)).get_output_in_json()
+        group_id = result[0]['properties']['groupId']
+
+        # Create a private endpoint connection
+        private_endpoint = self.cmd('network private-endpoint create -g {} -n {} --vnet-name {} --subnet {} -l {} '
```

Comment:
can you try to create with `--manual-request` so that it won't be approved automatically.

Reply:
This tests auto-approval workflow. will add tests for manual approval workflow in another PR.